src/RobotClient/RobotClient/Direction.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace RobotClient
+{
+    public class Direction
+    {
+        public readonly DateTime time;
+        public readonly double throttle;
+        public readonly double direction;
+
+        private readonly static string[] ThrottleStrings = { "Moving backwards", "In Neutral", "Moving forwards" };
+        private readonly static string[] DirectionStrings = { "and left", "", "and right" };
+
+        private Direction(DateTime time, double throttle, double direction)
+        {
+            this.time = time;
+            this.throttle = throttle;
+            this.direction = direction;
+        }
+
+        /**
+         * Parses a List of Directional data from the raw log string
+         */
+        public static List<Direction> ParseLog(string fullLogString)
+        {
+            var split = Regex.Split(fullLogString, Environment.NewLine);
```

Comment:
`Environment.NewLine` doesn't work because \n is written to the log but the windows newline is \r\n
```suggestion
            var split = Regex.Split(fullLogString, "\n");
```